pkg/scheduler/core/scheduling_queue.go

```diff
@@ -343,8 +448,15 @@ func isPodUpdated(oldPod, newPod *v1.Pod) bool {
 func (p *PriorityQueue) Update(oldPod, newPod *v1.Pod) error {
 	p.lock.Lock()
 	defer p.lock.Unlock()
+	_, exists, _ := p.activeQ.Get(newPod)
```

Comment:
I think we should use `oldPod` here and the line below. Although they should be practically the same given the current "key" function, but we might change the key function in the future.